observation/basic_health.py

```diff
@@ -63,23 +61,12 @@ class NoTargetsUpError(Exception):
         # the catalogue are ordered from highest to lowest priority)
         target = sources_above_horizon.targets[0]
         target.add_tags('bfcal single_accumulation')
-        session.standard_setup(**vars(opts))
+        session.capture_init()
+        user_logger.info("Only calling capture_start on correlator streams")
+        for correlator in session.cbf.correlators:
```

Comment:
Since starting data capture for all correlators would be the normal thing to do,  should we rather have the loop in session?

Reply:
I have a plan for this which involves passing the names of streams / instruments to `session.capture_start`. But that first needs another PR on katcorelib. I might sneak it into the current ska-sa/katcorelib#334...

Reply:
Also remember that the "normal" scripts simply call `session.capture_start()` and the subarray controller starts all streams. These scripts are not "normal".

Reply:
OK, reverted to starting all streams again. Whoohoo!

Reply:
> Also remember that the "normal" scripts simply call `session.capture_start()` and the subarray controller starts all streams. These scripts are not "normal".

`session.capture_start()` used to tell the subarray controller to start all the streams, but that changed in ska-sa/katcorelib/pull/327, so now the request goes directly to all data proxies that are part of the observation.

Reply:
True, I noticed that. I was considering optionally passing the streams to start as parameters to `session.capture_start` but that is tricky with things like FBFUSE, which I don't want the script to know about but it still needs to be started. For now we start them all again, making all the scripts more "normal" :-).